### ISSUE TITLE
Add support for separate power collectors

### ISSUE THREAD

**Issue by ledermann** [solectrus/forecast-collector#64]
As discussed in https://github.com/solectrus/hosting/issues/5:

- Allow to define two or more plants which individual plane declination, plane azimuth, and kWp (and maybe lat/long)
- Do separate API calls for each plant
- Sum up the result

Reference: https://github.com/jbaesner/quarkus-forecast-collector

**Comment by Bart1909** [comment]
Hi @ledermann,
maybe you can add a little instruction on how to use the quarkus-forecast-collector in combination with oolectrus?

**Comment by ledermann** [comment]
@Bart1909 There is an [.env example](https://github.com/jbaesner/quarkus-forecast-collector/blob/main/.env.example). Because I don't use the `quarkus-forecast-collector` by myself (we have just one roof), maybe @jbaesner as author can help.

**Comment by Bart1909** [comment]
Hi @ledermann,

I filled the .env file with my data and entered the influx settings vom solectrus. It seams, that this is working right now.
I've deactivated the solectrus forecast collector then.

Maybe @jbaesner can tell if this is the correct way.

Best regards

**Comment by jbaesner** [comment]
@Bart1909 yes, you can specify as many orientations as you wish. As can be seen in the [.env.example](https://github.com/jbaesner/quarkus-forecast-collector/blob/main/.env.example).

See for example:
```
FORECAST_CONFIGURATIONS=ROOF_1,ROOF_2

FORECAST_ROOF_1_LONGITUDE=13.3760
FORECAST_ROOF_1_LATITUDE=52.5191
FORECAST_ROOF_1_DECLINATION=30
FORECAST_ROOF_1_AZIMUTH=-180
FORECAST_ROOF_1_KWP=7

FORECAST_ROOF_2_LONGITUDE=13.3768
FORECAST_ROOF_2_LATITUDE=52.5186
FORECAST_ROOF_2_DECLINATION=30
FORECAST_ROOF_2_AZIMUTH=0
FORECAST_ROOF_2_KWP=8
```

**Comment by Bart1909** [comment]
> @Bart1909 yes, you can specify as many orientations as you wish. As can be seen in the [.env.example](https://github.com/jbaesner/quarkus-forecast-collector/blob/main/.env.example).
> 
> See for example:
> 
> ```
> FORECAST_CONFIGURATIONS=ROOF_1,ROOF_2
> 
> FORECAST_ROOF_1_LONGITUDE=13.3760
> FORECAST_ROOF_1_LATITUDE=52.5191
> FORECAST_ROOF_1_DECLINATION=30
> FORECAST_ROOF_1_AZIMUTH=-180
> FORECAST_ROOF_1_KWP=7
> 
> FORECAST_ROOF_2_LONGITUDE=13.3768
> FORECAST_ROOF_2_LATITUDE=52.5186
> FORECAST_ROOF_2_DECLINATION=30
> FORECAST_ROOF_2_AZIMUTH=0
> FORECAST_ROOF_2_KWP=8
> ```

Hi @jbaesner ,
okay and then setting the same influx settings as in solectrus is correct, right?
